cmd/fs/helpers.go

```diff
@@ -50,41 +51,34 @@ func filerForPath(ctx context.Context, fullPath string) (filer.Filer, string, er
 
 const dbfsPrefix string = "dbfs:/"
 const volumesPefix string = "dbfs:/Volumes"
+const localPefix string = "./"
```

Comment:
this won't work on Windows

Reply:
Finally got around testing this on Windows, it does seem to work:
<img width="798" alt="Screenshot 2024-08-02 at 9 34 56" src="https://github.com/user-attachments/assets/f8e8c995-e8d6-49c7-b21d-213404757272">

Reply:
....will also add support for .\

Reply:
Now .\ works too:
<img width="843" alt="Screenshot 2024-08-02 at 12 10 32" src="https://github.com/user-attachments/assets/344b6145-a428-4825-a109-b68ca0c7004e">